README.adoc

```diff
@@ -368,6 +368,41 @@ spec:
 
 The trigger for example filters the events by its type `ce-type=dev.knative.connector.event.timer`.
 
+== Secure Knative transports
+
+Knative brokers may use TLS encrypted transport options as described in https://knative.dev/docs/eventing/features/transport-encryption/#overview
+
+This means that Event producers need to use proper SSL authentication to connect to Https Knative broker endpoints with cluster-internal CA certificates.
+
+The IntegrationSource may use a volume mount with the cluster-internal CA certificates being injected.
+
+The integration source needs to enable the SSL client via environment variables and set the path to the injected CA certs and PEM files:
+
+* CAMEL_KNATIVE_CLIENT_SSL_ENABLED=true
+* CAMEL_KNATIVE_CLIENT_SSL_KEY_PATH=/knative-custom-certs/knative-eventing-bundle.pem
```

Comment:
`/knative-custom-certs/...` can potentially have multiple files https://knative.dev/docs/eventing/features/transport-encryption/#configure-custom-event-sources-to-trust-the-eventing-ca

Reply:
hm, this requires more sophisticated configuration then. Let's open a follow-up issue for supporting multiple files, ok?

Reply:
ok